src/erlcloud_aws.erl

```diff
@@ -1195,13 +1201,18 @@ canonical_headers(Headers) ->
 canonical_query_string([]) ->
     "";
 canonical_query_string(Params) ->
```

Comment:
please add a`-spec` to this function since it is now exported.

Reply:
done